src/work-allocation-2/models/cases/case.model.ts

```diff
@@ -0,0 +1,14 @@
+import CaseAction from './case-action.model';
+
+export default interface Case {
+  [key: string]: any;
+  id: string;
+  case_id: string;
```

Comment:
_Question:_
Why is this `snake_case` while all the other properties are `camelCase`?

Reply:
///